clients/go/cmd/zbctl/testdata/deploy.golden

```diff
@@ -5,13 +5,15 @@
       "bpmnProcessId": "process",
       "version": 1,
       "processDefinitionKey": "2251799813685345",
-      "resourceName": "model.bpmn"
+      "resourceName": "model.bpmn",
+      "tenantId": "<default>"`
```

Comment:
Updated this manually for now. I did not see any flag to update the golden files.